source/core/Date.cpp

```diff
@@ -349,7 +349,7 @@ namespace Vireo {
         struct tm tm;
         time_t timeVal = utcTime - kStdDT1970re1904;
         localtime_r(&timeVal, &tm);
-        _systemLocaleTimeZone = int(tm.tm_gmtoff);
+        _systemLocaleTimeZone = static_cast<int>(tm.tm_gmtoff);
```

Comment:
and here.

Reply:
Yes, but let's use the static_cast to keep the code using only one kind of casting.